css/style.css

```diff
@@ -425,6 +423,10 @@ html[xmlns] .clearfix {
 /* boxes */
 
 #boxes {
+  display: grid;

```

Comment:
The middleware box seems a bit out of place to me. Maybe we could center that if this looks good? It will need a media query though since it does not work on mobile.
`grid-column-start: 2;`
![Screenshot 2024-11-20 at 10 43 31 AM](https://github.com/user-attachments/assets/4555b300-3f31-4701-bc67-995bf64deff8)

Reply:
The grid looks nice. Also I like:

- The page text container does not flow to the page edges on large size
- the links are centred, esp on mobile.

Reply:
I am looking for a way to implement it

Reply:
Yeah could work. Just don't let it expand from 2 to 3?

Reply:
Maybe now?